frontend/src/model/beta/Settings.tsx

```diff
@@ -48,9 +50,9 @@ export default function Settings({ model }: SettingsProps) {
             <Typography variant='h6' component='h2'>
               Danger Zone!
             </Typography>
-            <Button variant='contained' disabled>
+            <LoadingButton variant='contained' disabled onClick={() => setLoading(true)} loading={loading}>
```

Comment:
Can we pull the handleClick logic into a separate function with a TODO? Makes it clearer what our intentions are in future.
```
const handleDelete = () => {
  setLoading(true)
  
  // TODO - Delete model API request and setLoading(false) on error
}

onClick={handleDeleteModel}
```